src/installer.rs

```diff
@@ -96,16 +96,34 @@ fn edition_compat_check(window: &gtk::Window, message: String) -> bool {
 }
 
 fn connectivity_check(window: &gtk::Window, message: String) -> bool {
-    let status = match reqwest::blocking::get("https://cachyos.org") {
+    // First try HTTP check to cachyos.org
+    let http_status = match reqwest::blocking::get("https://cachyos.org") {
         Ok(resp) => resp.status().is_success() || resp.status().is_server_error(),
         _ => false,
     };
 
-    if !status {
-        utils::show_simple_dialog(window, gtk::MessageType::Error, &fl!("offline-error"), message);
-        return false;
+    if http_status {
+        return true;
     }
-    true
+
+    // If HTTP check fails, try ping fallback to reliable DNS servers
+    
+    for target in ["8.8.8.8", "1.1.1.1", "9.9.9.9"] {
```

Comment:
```suggestion
    for target in ["2001:4860:4860::8888", "2606:4700:4700::1111", "2620:fe::fe", "8.8.8.8", "1.1.1.1", "9.9.9.9"] {
```

Add IPv6 targets and prefer them.